tests/Lykke.RabbitMq.Azure.Tests/MigrationRepositoryBlobPublishingQueueRepositoryTest.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using AzureStorage.Blob;
+using Lykke.RabbitMqBroker.Publisher;
+using Lykke.RabbitMqBroker.Subscriber;
+using NUnit.Framework;
+
+namespace Lykke.RabbitMq.Azure.Tests
+{
+    [Obsolete("Used only for migration test")]
+    [TestFixture]
```

Comment:
I would better used an Explicit tag with a description instead of Obsolete.

Reply:
Agree